### PR TITLE
feat: added toast and loader

### PR DESCRIPTION
**Pull request by sri-rishi**

### CODE REVIEW

File: src/Components/Loader/loader.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	animation-name: rotatepos;
+	animation-duration: 2s;
+	animation-iteration-count: infinite;
+}
+.inside1{
+	display: flex;
+	place-items: center;
+	justify-content: center;
+	width: 110px;
+	height: 110px;
```
Comment (yaswanthmyneni) [inline src/Components/Loader/loader.css:17]: changes: use relative units like rems for more than 16px

File: src/Components/Toast/toast.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.toast {
+    position: fixed;
+    top: 8rem;
+    right: 2rem;
+    transition: transform .6s ease-in-out;
+    animation: toast-in-right .7s;
+    gap: 1rem;
+    z-index: 99999;
```
Comment (yaswanthmyneni) [inline src/Components/Toast/toast.css:8]: changes: don't give huge numbers for z-index and also use CSS variables for z-index

**Review by yaswanthmyneni — APPROVED** [review]
you can merge after changes

File: src/api-calls/deleteMethodApiCalls.js [hunk 3]
```diff
@@ -1,23 +1,38 @@
...
 }
 
-const deleteFromWishList = async(id, wishistDispatch) => {
+const deleteFromWishList = async(id, wishistDispatch, setToastData) => {
     const token = localStorage.getItem("token")
     try {
         const response = await axios.delete(`/api/user/wishlist/${id}`, {headers:{authorization: token}});
         if((response.status === 201 || response.status === 200)) {
             wishistDispatch({type: "SET_WISHLIST", payload: response.data.wishlist});
+            setToastData({
```
Comment (Shivani-1524) [inline src/api-calls/deleteMethodApiCalls.js:30]: nice use of objects

File: src/Components/Loader/loader.css [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+	border-left: 20px solid var(--darker-yellow-color);
+	border-top: 20px solid transparent;
+	border-bottom: 20px solid transparent;
+	border-right: 20px solid var(--darker-yellow-color);
+	animation-name: buruh;
+	animation-duration: 2s;
+	animation-iteration-count: infinite;
+}
+.loader1{
+	height: 150px;
```
Comment (Shivani-1524) [inline src/Components/Loader/loader.css:28]: use rems here

File: src/Components/Loader/loader.css [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+.inside1{
+	display: flex;
+	place-items: center;
+	justify-content: center;
+	width: 110px;
+	height: 110px;
+	border-radius: 50%;
+	border-left: 20px solid var(--darker-yellow-color);
+	border-top: 20px solid transparent;
+	border-bottom: 20px solid transparent;
```
Comment (Shivani-1524) [inline src/Components/Loader/loader.css:21]: why use transparent borders when margin or padding would give the same effect?

File: src/Components/Loader/loader.css [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+	border-bottom: 20px solid var(--darker-yellow-color);
+	animation-name: rotatepos;
+	animation-duration: 2s;
+	animation-iteration-count: infinite;
+}
+.inside1{
+	display: flex;
+	place-items: center;
+	justify-content: center;
+	width: 110px;
```
Comment (Shivani-1524) [inline src/Components/Loader/loader.css:16]: use rems here

**Review by Shivani-1524 — COMMENTED** [review]
code looks good to merge after the changes

**Pull request merged**